packages/e2e-test-utils-playwright/src/editor/site-editor.ts

```diff
@@ -16,7 +16,8 @@ export async function saveSiteEditorEntities( this: Editor ) {
 	await this.page.click(
 		'role=region[name="Save panel"i] >> role=button[name="Save"i]'
 	);
-	await this.page.waitForSelector(
-		'role=region[name="Editor top bar"i] >> role=button[name="Saved"i][disabled]'
-	);
+	// A role selector cannot be used here because it needs to check that the `is-busy` class is not present.
+	await this.page.waitForSelector( '[aria-label="Saved"].is-busy', {
```

Comment:
Could you elaborate a bit more on why the `is-busy` class is needed here? Is `disabled` or `aria-disabled` not sufficient for checking if it's saving? Judging from the code at:

https://github.com/WordPress/gutenberg/blob/eef6fe95618b45a442b72850ca794caef1feb825/packages/edit-site/src/components/save-button/index.js#L38

, it seems like they should be updated at the same time. 🤔

Reply:
What it is trying to check is when it has finished saving. It needs to wait for that before navigating to the homepage. From what I saw, there are three different states:

* Not saved: `disabled` is **false**. 
* Saving: `disabled` is **true** and label is "Saved".
* Saved: `disabled` is still **true** and label is also "Saved". You can't interact with the button until you modify the content again.

I didn't find another way to differentiate between Saving/Saved without using the class.

Reply:
Seems like a bug in the site editor to me? Why does it show "Saved" when it's not actually saved?